packages/esm-ward-app/src/ward-view-header/admission-requests-bar.component.tsx

```diff
@@ -31,9 +38,10 @@ const AdmissionRequestsBar: React.FC<AdmissionRequestsBarProps> = ({ wardPending
   }
 
   return (
-    <div className={`${styles.admissionRequestsContainer} ${
-      inpatientRequests?.length ? styles.blackBackground : styles.lightBlueBackground
-    }`}>
+    <div
+      className={`${styles.admissionRequestsContainer} ${
+        inpatientRequests?.length ? styles.blackBackground : styles.lightBlueBackground
+      }`}>
```

Comment:
nit: Prefer using `classnames` here.

Reply:
@denniskigen  Kindly, I am somehow confused. This is not part of my changes, though I don't mind fixing the issue. I made changes on one file and would like to know why I have 8 files changed?

Reply:
Looks like changes from linting